packages/camlidl/camlidl.1.10/opam

```diff
@@ -26,6 +26,9 @@ extra-files: [
   ["camlidl.install" "md5=cf56e14faed046880b7c9d0f4cd737f1"]
   ["META" "md5=8ea7ab71c0457962a6382e94c2b5b8bc"]
 ]
+conflicts: [
+  "ocaml-option-bytecode-only" { ocaml:version >= "5.0" }
```

Comment:
```suggestion
  "ocaml-option-bytecode-only"
```

Reply:
This is for all bytecode-only compiler versions. Tested locally with
```
opam switch create  ocaml.4.14.1+options ocaml-option-bytecode-only
```
and got the failure
```
#=== ERROR while compiling camlidl.1.11 =======================================#
# context     2.1.3 | macos/arm64 | ocaml-option-bytecode-only.1 | git+https://github.com/ocaml/opam-repository.git
# path        /Volumes/ext/.opam/ocaml.4.14.1+options/.opam-switch/build/camlidl.1.11
# command     /Volumes/ext/.opam/opam-init/hooks/sandbox.sh build make all
# exit-code   2
# env-file    /Volumes/ext/.opam/log/camlidl-14325-e3ada1.env
# output-file /Volumes/ext/.opam/log/camlidl-14325-e3ada1.out
### output ###
# [...]
# 11 warnings generated.
# ocamlc -g -ccopt "-Wall -g" comerror.c
# rm -f dllcamlidl.so
# ocamlmklib -o camlidl  idlalloc.o comintf.o comerror.o
# cd lib; /Library/Developer/CommandLineTools/usr/bin/make all
# ocamlc -g -c com.mli
# ocamlc -g -c com.ml
# ocamlc -g -a -o com.cma -dllib -lcamlidl -cclib -lcamlidl com.cmo
# ocamlopt -c com.ml
# make[1]: ocamlopt: No such file or directory
# make[1]: *** [com.cmx] Error 1
# make: *** [all] Error 2
```